JFS_CAN_report_instructions.qmd

```diff
@@ -0,0 +1,31 @@
+---
+title: "JFS CAN Data Report Instructions"
+format: pdf
+editor: visual
+---
+
+## Open the Starter App
+
+1.  In Docker Desktop, search for "Degauss/shiny_starter"
+
+    A. "Pull" the container
+
+2.  Once the container is in your Desktop, find it and hit "Run"
+
+    A. In the popup box, expand the options and in the space marked "Local Host", type in "3838" and hit Run
+
+3.  Navigate to the Containers page and in your line for the shiny_starter, click to button to open in browser
+
+4.  Now that you're in the app, you can upload your .csv file
+
+5.  After the upload, the app will show you a sample of your original file and the prepared file for use in the next program.
+
+6.  Click the button to download the prepped file
```

Comment:
I think what happens when you click download will be different depending on your browser's settings. I have set to "always ask..." for downloads in Firefox, so I have to navigate to where I want to save it.  Let's add a step that ensure the user will save the file to a known location (e.g., `~/Downloads`) that they can navigate to in the next step.